code/modules/jobs/job_types/detective.dm

```diff
@@ -11,7 +11,7 @@
 	selection_color = "#ffeeee"
 	chat_color = "#C78B8B"
 	minimal_player_age = 7
-	exp_requirements = 180
+	exp_requirements = 900
```

Comment:
wew 15 hours of sec is a long time

Reply:
Again this is intended to be 15 hours as crew, not sec. This is just entry level for sec, and detective should not have any lower of a rec than normal officers. 

Look at the variables on cyborg.dm?